src/plugins/blocksmith/components/imgur-post-block-preview/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import ImgurPostBlockEmbed from './embed';
+
+class ImgurPostBlockPreview extends Component {
+  constructor(props) {
+    super(props);
+    this.createEmbed = this.createEmbed.bind(this);
+    this.loadScript = this.loadScript.bind(this);
+  }
+
+  loadScript() {
+    const embedScript = document.getElementById('globalImgurEmbedScriptTagId');
+
+    if (embedScript) {
+      // eslint-disable-next-line no-unused-expressions
+      (embedScript.parentElement) ? embedScript.parentElement.removeChild(embedScript) : null;
```

Comment:
@jalopez13 we dont do ternaries like this, just do 

```
if (embedScript.parentElement) {
  theThing()
}
```

Reply:
@efjacobson 

Removed ternary and updated conditional.